hack/e2e-test.sh

```diff
@@ -34,15 +34,18 @@ function cleanup {
         kubectl logs -n kube-system kube-controller-manager-kind-control-plane > $ARTIFACTS/kube-controller-manager.log || true
         kubectl logs -n kueue-system deployment/kueue-controller-manager > $ARTIFACTS/kueue-controller-manager.log || true
         kubectl describe pods -n kueue-system > $ARTIFACTS/kueue-system-pods.log || true
-        $KIND delete cluster --name $KIND_CLUSTER_NAME || { echo "You need to run make kind-image-build before this script"; exit -1; }
+        $KIND delete cluster --name $KIND_CLUSTER_NAME -v 5
```

Comment:
Why do we need to increase verbose? Do you have any cases you want to increase this verbose?

Reply:
My concern is to overlook important logs by this increase.

Reply:
there war some cases when the kind cluster did not start. https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/913/pull-kueue-test-e2e-main-1-25/1674051550193913856

Reply:
I guess it's not worth during delete, just create?

Reply:
done